openreview/stages/venue_stages.py

```diff
@@ -1322,6 +1322,7 @@ class ReplyTo(Enum):
         WITHFORUM = 1
         REVIEWS = 2
         METAREVIEWS = 3
+        REVIEW_REVISIONS = 4
```

Comment:
I'm wondering if this should be another parameter to specify if the custom stage should "replies" or "edits" on the ReplyTo selected value.
This way you can use the custom stage to create edits for reviews, metareviews and forums. I know that the SubmissionRevisionStage already works for forums...

Reply:
Yes, I thought about this. I think the CustomStage is getting more and more complicated and it might not be easy for PCs to know exactly how to do what they need. But it does make sense to do this. I'll make this change.

Reply:
I agree that is becoming more complex, maybe we should have a CustomStage for making replies and another CustomStage for editing existing notes.

Reply:
just saw this message! If we think the current code is too complex, I can separate the CustomStage into two.